code/src/components/AgeQ.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+const AgeQ = ({
```

Comment:
I would consider calling this component AgeQuestion, to be super clear and descriptive :)